test/utils/index.ts

```diff
@@ -13,12 +12,12 @@ export function sleep(ms: number) {
 
 export async function verifyEnv(env: LuminoTestEnvironment): Promise<void> {
```

Comment:
if the bootstrapping test is being removed, i don't see any reason to keep this function around

Reply:
Agree